frontend/src/app/commonComponents/TextWithTooltip.tsx

```diff
@@ -0,0 +1,41 @@
+import { Tooltip } from "@trussworks/react-uswds";
+import React from "react";
+
+type CustomButtonProps = React.PropsWithChildren<{
+  className?: string;
+}> &
+  JSX.IntrinsicElements["button"] &
+  React.RefAttributes<HTMLButtonElement>;
+
+interface Props {
+  text: string;
+  tooltip: string;
+  position?: "top" | "bottom" | "left" | "right";
+}
+
+export const TextWithTooltip = ({ text, tooltip, position }: Props) => {
```

Comment:
**ask:** Could you create a story for this tooltip component?

Reply:
Added!